.storybook/decorators/contextsWrapper.js

```diff
@@ -53,7 +53,7 @@ export const withContextWrapper = makeDecorator({
 			// Change background color when demonstrating static color options.
 			if (args.staticColor === "black") {
 				container.style.background = "var(--spectrum-examples-gradient-static-black)";
-			} else if (args.staticColor === "white") {
+			} else if (args.staticColor === "white" || args.isStaticWhite === "true") {
```

Comment:
In the case of Progress Bar, only Static White is supported. I didn't want to make folks use a dropdown in the Storybook controls when there's only one option, so I expanded this to accommodate for that